### DISCUSSION TITLE
Recover 10 year old account!

### DISCUSSION THREAD

**Discussion by xardit** [community/community#70521] [Other Feature Feedback, Questions, & Ideas]
### How to recover 10 year old account?

I have an old account that I tried to recover but don't have email or password.
I expected it to expire since it has zero activity from the start but never happened! 
What does the github do for old accounts?

**Comment by ghostinhershell** [comment]
Hey @a3diti, thanks so much for reaching out in the community. It's possible that you _may_ be able to re-gain access to your old account but you will need to reach out to support in order to confirm that. 

As a rule of thumb, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
